packages/block-editor/src/components/url-popover/link-viewer.js

```diff
@@ -9,7 +9,6 @@ import classnames from 'classnames';
 import { __ } from '@wordpress/i18n';
 import { ExternalLink, Button } from '@wordpress/components';
 import { safeDecodeURI, filterURLForDisplay } from '@wordpress/url';
-import { pencil } from '@wordpress/icons';
```

Comment:
Do you need the changes in this file anymore?

Reply:
no.